app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -993,16 +993,21 @@ def build_toolbar_disable_button(id)
       when "host_restart"
         return @record.is_available_now_error_message(:reboot) if @record.is_available_now_error_message(:reboot)
       end
-    when "ManageIQ::Providers::Kubernetes::ContainerManager::ContainerNode"
+    when "ContainerNode"
```

Comment:
@matthewd I suppose these were "automatically" changed but it seems that the spec and the infrastructure can't handle the new namespaces.